src/main/java/com/github/jreddit/oauth/param/RedditTokenScopes.java

```diff
@@ -0,0 +1,50 @@
+package com.github.jreddit.oauth.param;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Manager of the scopes a response from reddit returns.
+ * Used by <i>RedditToken</i> to parse the list of scopes it receives.
+ * 
+ * @see RedditToken
+ * 
+ * @author Simon Kassing
+ */
+public class RedditTokenScopes {
```

Comment:
can we ditch the plural of this class or renaming in something else. It's against best practices to name in plural classes.